back/CinemaReservation.Database/CinemaReservation.Database.sqlproj

```diff
@@ -61,6 +61,7 @@
     <Folder Include="Properties" />
     <Folder Include="dbo\" />
     <Folder Include="dbo\Tables\" />
+    <Folder Include="StoredProcedures" />
```

Comment:
dbo\StoredPrecedures because SPs also has schema. Also all this Folders can be excluded from project file when some files are added